### DISCUSSION TITLE
Drag & Drop Upload - User error or bug?

### DISCUSSION THREAD

**Discussion by GinjNTonic** [ciur/papermerge#549] [General]
I wasn't sure I should post this in issues in case it was user error, so apologies if this isn't the ideal place for this.  

I'm not able to upload single or multiple files by drag and drop.  I checked the documentation and I seem to be doing it as the example gif shows.  I'm not attempting to upload folders.  Instead of uploading the PDF opens in my browser.  

DETAILS:
- Installed today by docker-compose using the ["Complete Stack in 2 mins" template](https://raw.githubusercontent.com/papermerge/papermerge-core/master/docker/docker-compose.yml) in Portainer running on OMV6.
- Server is lan access only using http for now.
- Tried 3 different browsers: LibreWolf, Brave & Firefox with the same result.
- My client PC is using Dolphin file manager.
- Manual upload by clicking the upload button works fine.
- [Edit to add] The PDFs are NOT password protected

It appears to be working great so far other than this.  Looks like a great app that'll be really useful to me, so thank you!

**Comment by ciur** [comment]
yes, that is a bug!
